thruster-control.h

```diff
@@ -6,18 +6,21 @@ struct __WHICHAMI__ {
     char name[25];   // Note that this is static, and will not be allocated dynamically.
     int data_source;
     int data_send;
+    int fd;
 } Whichami;
 
+//default is 0x40...
 // TODO: these definitions act as names for our thrusters. What values should they hold (i2c addresses or whatev)
-#define T_H_FRONTLEFT  0
-#define T_H_FRONTRIGHT 0
-#define T_H_BACKLEFT   0
-#define T_H_BACKRIGHT  0
-#define T_V_LEFT       0
-#define T_V_RIGHT      0
-#define T_V_FRONT      0
-#define T_V_BACK       0
+#define T_H_FRONTLEFT  0x40
+#define T_H_FRONTRIGHT 0x40
+#define T_H_BACKLEFT   0x40
+#define T_H_BACKRIGHT  0x40
+#define T_V_LEFT       0x40
+#define T_V_RIGHT      0x40
+#define T_V_FRONT      0x40
+#define T_V_BACK       0x40
 
+//need API stuff for this @Chris
```

Comment:
API stuff is on me and will be ready very soon, what needs to be here?

Reply:
idk you said that the ports would be API ports